api/src/main/kotlin/gropius/schema/mutation/ArchitectureMutations.kt

```diff
@@ -0,0 +1,483 @@
+    ): Project {
+        return projectService.updateProject(dfe.gropiusAuthorizationContext, input)
+    }
+
+    @GraphQLDescription("Deletes the specified Project, requires ADMIN on the project to delete")
+    @AutoPayloadType("The id of the deleted Project")
+    suspend fun deleteProject(
+        @GraphQLDescription("Defines which Project to delete")
+        input: DeleteNodeInput, dfe: DataFetchingEnvironment
+    ): ID {
+        projectService.deleteProject(dfe.gropiusAuthorizationContext, input)
+        return input.id
+    }
+
+    @GraphQLDescription("Creates a new InterfaceSpecification, requires ADMIN on the Component.")
+    @AutoPayloadType("The created InterfaceSpecification")
+    suspend fun createInterfaceSpecification(
+        @GraphQLDescription("Defines the created InterfaceSpecification")
+        input: CreateInterfaceSpecificationInput, dfe: DataFetchingEnvironment
+    ): InterfaceSpecification {
+        return interfaceSpecificationService.createInterfaceSpecification(dfe.gropiusAuthorizationContext, input)
+    }
+
+    @GraphQLDescription("Updates the specified InterfaceSpecification, requires ADMIN on the Component of the InterfaceSpecification to update")
+    @AutoPayloadType("The updated InterfaceSpecification")
+    suspend fun updateInterfaceSpecification(
+        @GraphQLDescription("Defines which InterfaceSpecification to update and how to update it")
+        input: UpdateInterfaceSpecificationInput, dfe: DataFetchingEnvironment
+    ): InterfaceSpecification {
+        return interfaceSpecificationService.updateInterfaceSpecification(dfe.gropiusAuthorizationContext, input)
+    }
+
+    @GraphQLDescription("Deletes the specified InterfaceSpecification, requires ADMIN on the Component of the InterfaceSpecification to delete")
+    @AutoPayloadType("The id of the deleted InterfaceSpecification")
+    suspend fun deleteInterfaceSpecification(
+        @GraphQLDescription("Defines which InterfaceSpecification to delete")
+        input: DeleteNodeInput, dfe: DataFetchingEnvironment
+    ): ID {
+        interfaceSpecificationService.deleteInterfaceSpecification(dfe.gropiusAuthorizationContext, input)
+        return input.id
+    }
+
+    @GraphQLDescription("Creates a new InterfaceSpecificationVersion, requires ADMIN on the Component.")
+    @AutoPayloadType("The created InterfaceSpecificationVersion")
+    suspend fun createInterfaceSpecificationVersion(
+        @GraphQLDescription("Defines the created InterfaceSpecificationVersion")
+        input: CreateInterfaceSpecificationVersionInput, dfe: DataFetchingEnvironment
+    ): InterfaceSpecificationVersion {
+        return interfaceSpecificationVersionService.createInterfaceSpecificationVersion(
+            dfe.gropiusAuthorizationContext, input
+        )
+    }
+
+    @GraphQLDescription("Updates the specified InterfaceSpecificationVersion, requires ADMIN on the Component of the InterfaceSpecificationVersion to update")
```

Comment:
ohm what? since when is admin inherited from the InterfaceDefinition?  however, still improved doc a bit with 0cafffbb2e0d25190191a14ecced0ed02e5389ad due to copy-past-error